alyssa_mckee/flask_fundamentals/Ninja_Gold/templates/index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Ninja Money</title>
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}">
+	<!-- <link rel="stylesheet" type="text/css" href="../static/styles.css"> -->
+</head>
+<body>
+	<h1>Ninja Gold Game</h1>
+	<div id="head">
+		<h3>your gold: <span>{{ gold }}g</span></h3>
+	</div>
+
+	<form action="/process_money" method="POST">
```

Comment:
Very cool way to handle multiple buttons that go to the same location. Good job!